src/ensembl/src/content/app/about/components/about-menu/AboutMenu.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Menu as MenuType } from 'src/shared/types/help-and-docs/menu';
+import { MenuItem } from 'src/shared/types/help-and-docs/menu';
+
+import styles from './AboutMenu.scss';
+
+type Props = {
+  menu: MenuType;
+  currentUrl: string;
+};
+
+export const TopMenu = (props: Props) => {
+  const { menu } = props;
+
+  const menuItems = menu.items;
```

Comment:
Do we need to create this variable? In line 55 `menu.items` is used without assigning it to to a new variable/constant.

Reply:
🤷‍♂️  ✅